api/jobs-api/src/main/java/com/cognifide/aet/job/api/collector/JsErrorLog.java

```diff
@@ -19,12 +19,9 @@
 import java.util.Objects;
 import org.apache.commons.lang3.builder.CompareToBuilder;
 
-/**
- * @author lukasz.wieczorek
- */
 public class JsErrorLog implements Serializable, Comparable<JsErrorLog> {
 
-  private static final long serialVersionUID = -8929508257448156720L;
+  private static final long serialVersionUID = -8438978831466054811L;
```

Comment:
I think there is no need to change the `serialVersionUID`
Additionally: such change would break backward-compatibility in Maven plugin.

Please revert his line.

Reply:
ok